src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -127,7 +127,7 @@ public void ValidPathWithTrailingSlash()
         }
 
         [ConditionalFact(nameof(UsingNewNormalization))]
-        [SkipOnTargetFramework(Tfm.BelowNet462 | Tfm.Core50, "long path support added in 4.6.2")]
+        [SkipOnTargetFramework(Tfm.Core50)]
```

Comment:
I guess that means removing C:\src\System.IO.FileSystem\tests\Tfm.cs entirely. Possibly in a future PR.